lib/Backend/Lower.cpp

```diff
@@ -9040,6 +9038,30 @@ Lowerer::LowerStElemI(IR::Instr * instr, Js::PropertyOperationFlags flags, bool
         helperMethod == IR::HelperOp_InitClassMemberSetComputedName
         );
 
+    Assert(dst->IsIndirOpnd());
+    IR::IndirOpnd* dstIndirOpnd = dst->AsIndirOpnd();
+
+    if (helperMethod == IR::HelperOP_InitElemGetter || helperMethod == IR::HelperOP_InitElemSetter)
```

Comment:
I think we might have the same problem with `HelperOp_InitClassMemberGetComputedName` and `HelperOp_InitClassMemberSetComputedName`. These methods call `Op_InitElemGetter` and `Op_InitElemSetter` respectively.

Reply:
Interesting,
Paolo, could you try to see if the bug repro with
```js
function foo() {
  c = class {
    get [y]() {}
    set [z](s) {}
  }
}
foo();
foo();
console.log(typeof (new c())[1])
console.log(typeof (new c())[2])
```

Reply:
Yes, it is also broken; output: 
function
function